.rubocop_cocoapods.yml

```diff
@@ -14,6 +14,9 @@ Documentation:
 
 #- CocoaPods -----------------------------------------------------------------#
 
+ParameterLists:
```

Comment:
I moved this from .rubocop.yml because it was conflicting with .rubocop_todo.yml, where else should it be updated?